### PR TITLE
Optimize material-ui imports

### PR DESCRIPTION
**Pull request by diboy2**

### CODE REVIEW

File: client/next.config.js [hunk 1]
```diff
@@ -1,4 +1,17 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
+  experimental: {
```
Comment (demiankatz) [inline client/next.config.js]: Is there a roadmap for this functionality to become non-experimental? Right now this is triggering a "use at your own risk" warning.

**Review by demiankatz — COMMENTED** [review]
This doesn't seem to break anything, but I'm a little concerned about using an experimental feature if we can avoid that (see below). I also think there are going to be some conflicts to resolve after you merge #118.

File: client/next.config.js (hunk 1 above)
Comment (diboy2) [reply client/next.config.js]: I decided against this approach. I'm going to prevent incorrect import styles through eslint.

**Review by demiankatz — APPROVED** [review]
This makes sense to me -- thanks!

**Pull request merged**